### PR TITLE
Update action dispute link to new Gardens uri scheme

### PR DESCRIPTION
**Pull request by Corantin**

### CODE REVIEW

**Review by felixbbertrand — APPROVED** [review]

**Comment by felixbbertrand** [comment]
CI issue has to be resolved soon

File: src/disputables/known-arbitrables.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const KnownArbitrables = {
       {
         address: '0x59a15718992a42082ab2306bc6cbd662958a178c',
         urlBuilder: (actionId, entityPath) =>
-          `https://1hive.org/#/${entityPath}/${actionId}`,
+          `https://gardens.1hive.org/#/xdai/garden/0x8ccbeab14b5ac4a431fffc39f4bec4089020a155/${entityPath}/${actionId}`,
```
Comment (0xGabi) [inline src/disputables/known-arbitrables.js:10]: Note we should also consider the network as new parm here since now Celeste is a multichain website.

**Comment by Corantin** [comment]
Right I'll make a fix

File: src/disputables/known-arbitrables.js (hunk 1 above)
Comment (Corantin) [reply src/disputables/known-arbitrables.js:10]: But isn't a config for xdai only anyway ?

Comment (Corantin) [reply src/disputables/known-arbitrables.js:10]: If we want to have it for rinkeby also we will have to put it in the rinkeby config just above xdai

**Pull request merged**